modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/components/_field-options.scss

```diff
@@ -41,9 +41,13 @@ $c: '.ddm-field-options';
 				background: transparent;
 				border: none;
 				color: #6b6c7e;
-				cursor: default;
+				cursor: auto;
```

Comment:
@alinedoleron, `auto` is the default value of the `cursor` property, right? Therefore, we can remove this line. 😉 

In addition, why didn't you squash this commit with the first one? Let's commit less putting together very related commits, ok?